Zotero/Scenes/Detail/PDF/Views/PDFDocumentViewController.swift

```diff
@@ -352,6 +366,10 @@ final class PDFDocumentViewController: UIViewController {
         stateManager.lineWidth = lineWidth
     }
 
+    func setInterface(hidden: Bool) {
+        self.pdfController?.userInterfaceView.alpha = hidden ? 0 : 1
```

Comment:
PSPDFKit uses `alpha` instead of `isHidden` internally, so if I restore hidden ui state by setting `isHidden` the UI just never shows up after that. It surprised me too.